README.md

```diff
@@ -17,7 +17,7 @@ codedoc install
 codedoc serve
 ```
 
-By default, your blog will be served on `localhost:3000/coding-blog-boilerplate`. Checkout [**CODEDOC** documentation](https://codedoc.cc) to learn how to change that (or generally work with **CODEDOC**).
+By default, your blog will be served on [localhost:3000/coding-blog-boilerplate](http://localhost:3000/coding-blog-boilerplate). Checkout [**CODEDOC** documentation](https://codedoc.cc) to learn how to change that (or generally work with **CODEDOC**).
```

Comment:
well that link isn't always the correct link. it is only the correct one by default, but with other codedoc configurations it would not (and this would result in a broken link).

Reply:
True, but I still think it is most helpful to "optimize" for the default case.  This `README.md` is really only temporary documentation anyway.  Eventually I would expect that users replace all its contents with information about their blog written to the readers of their blog.  That is certainly what I will do.